doc/release/0.4.rst

```diff
@@ -15,6 +15,7 @@ New Features
    :func:`armi.utils.hexagon.getIndexOfRotatedCell`
    (`PR#1846 <https://github.com/terrapower/armi/1846`)
 #. Allow merging a component with zero area into another component (`PR#1858 <https://github.com/terrapower/armi/pull/1858>`_)
+#. Use inner diameter for sorting components when outer diameter is identical (`PR#1882 <https://github.com/terrapower/armi/pull/1882>`_)
```

Comment:
```suggestion
#. Allow merging a component with zero area into another component. (`PR#1858 <https://github.com/terrapower/armi/pull/1858>`_)
#. Use inner diameter for sorting components when outer diameter is identical. (`PR#1882 <https://github.com/terrapower/armi/pull/1882>`_)
```

Reply:
Arrielle pointed out to me last release that these periods greatly improve how the release docs render.

So I'm just adding them back in. NBD